package.json

```diff
@@ -6,9 +6,7 @@
   "devDependencies": {
     "github-release-notes": "^0.17.1",
     "paluh-litps": "^0.1.4",
-    "pulp": "^15.0.0",
-    "purescript": "^0.14.0",
-    "spago": "^0.19.1"
```

Comment:
Hey @sigma-andex why did you remove these two dependencies from here?

Reply:
Updated